week3/assignment/src/components/ByRandom/RandomResult.jsx

```diff
@@ -0,0 +1,49 @@
+import DATA from "../../assets/DATA";
+import styled from "styled-components";
+import { Question, SectionWrapper } from "../../style/MainStyle";
+
+const RandomResult = (props) => {
+  // eslint-disable-next-line react/prop-types
+  const { isStart, isStartHandler } = props;
+  const randomIdx = Math.floor(Math.random() * DATA.length);
+  const AlbumCover = DATA[randomIdx].img;
+  const SongTitle = DATA[randomIdx].title;
+
+  return (
+    <>
+      <SectionWrapper>
+        <Question>오늘의 추천 노래는 바로!</Question>
+        <Img src={AlbumCover} alt="랜덤 노래 커버 이미지"></Img>
+        <Title>{SongTitle}</Title>
```

Comment:
이렇게 되면 `AlbumCover` 대신 `shuffledArray[0].img` 이런식으로 바꿀 수 있을 것 같아요!
아니면,

```suggestion
    let randomImg = shffledArray[0]
```
이렇게 선언하고 들어가도 딜듯?